tests/components/overseerr/test_sensor.py

```diff
@@ -0,0 +1,33 @@
+"""Test the Overseerr sensor platform."""
+import pytest
+from syrupy.assertion import SnapshotAssertion
+from syrupy.filters import props
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+
+@pytest.mark.usefixtures("setup_integration")
+@pytest.mark.parametrize(
+    "sensor_name",
+    [
+        "movie_requests",
+        "tv_requests",
+        "approved_requests",
+        "available_requests",
+        "pending_requests",
+        "total_requests",
+    ],
+)
```

Comment:
the idea is correct, but please use a loop here (because it's quicker). You'll need to give your snapshots a name like `snapshot(name=f"entry-{key}")` and `snapshot(name=f"state-{key}")` for that to work.

Reply:
Done! (I hope, not used to pytest and snapshot testing)